stories/02-configuration/04-WordPress.stories.mdx

```diff
@@ -3,19 +3,94 @@
 # WordPress Configurations
 This section outlines the different ways you can configure WP Irving.
 
-Coming soon.
-
 ## Available Hooks
-This section is a work in progress.
+**Disclaimer: This section is a work in progress. Hooks will rarely be removed, but this list is not yet comprehensive.**
```

Comment:
We should look at dynamically generating these docs. I'm not sure how much value a list of hooks without usage info is in this context.

Reply:
Agreed, I'll create a ticket for it. If anything, it's more of an acknowledgment that they exist and putting up a big flag for us to make sure we come back to it.